WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -66,7 +91,6 @@ private extension PointOfSaleDashboardView {
     var cartView: some View {
         CartView(viewModel: viewModel,
```

Comment:
We probably also need to update **CartView** and **ItemRowView** since the button is pushing the labels and not allowing them to be in one line:

<img width="409" alt="Screenshot 2024-07-04 at 09 59 22" src="https://github.com/woocommerce/woocommerce-ios/assets/6242034/126c6815-c32a-4c72-bd34-d27d774616ca">

Reply:
Good catch, we can handle this separately when also dealing with product names and prices "too long for the available space". Logged on: https://github.com/woocommerce/woocommerce-ios/issues/13262